code/modules/jobs/job_types/roguetown/Inquisition/puritan.dm

```diff
@@ -25,9 +26,163 @@
 	jobtype = /datum/job/roguetown/puritan
 	allowed_patrons = list(/datum/patron/old_god)
 
-/datum/outfit/job/roguetown/puritan/pre_equip(mob/living/carbon/human/H)
+/datum/job/roguetown/puritan/after_spawn(mob/living/L, mob/M, latejoin = TRUE)
+	. = ..()
+	if(ishuman(L))
+		var/mob/living/carbon/human/H = L
+		H.advsetup = 1
+		H.invisibility = INVISIBILITY_MAXIMUM
+		H.become_blind("advsetup")
+
+
+////Classic Inquisitor with a much more underground twist. Use listening devices, sneak into places to gather evidence, track down suspicious individuals. Has relatively the same utility stats as Confessor, but fulfills a different niche in terms of their combative job as the head honcho. 
+
+/datum/advclass/puritan/inspector
+	name = "The Eye of Psydon"
+	tutorial = "Your eternal service to fighting the FORCES OF HERESY has manifested within you into a profession of underground connections of Confessors, away from the prying eyes of others. Stop HERESY in their tracks, predict their movements, collect evidence through listening devices and, when the time comes - strike with impunity with your blade and swift dodging prowess."
+	outfit = /datum/outfit/job/roguetown/puritan/inspector
+
+	category_tags = list(CTAG_PURITAN)
+
+/datum/outfit/job/roguetown/puritan/inspector/pre_equip(mob/living/carbon/human/H)
 	..()
 	shirt = /obj/item/clothing/suit/roguetown/shirt/undershirt/puritan
+	armor = /obj/item/clothing/suit/roguetown/armor/leather/heavy/coat
+	belt = /obj/item/storage/belt/rogue/leather/knifebelt/black/psydon
+	neck = /obj/item/clothing/neck/roguetown/psicross/silver
+	shoes = /obj/item/clothing/shoes/roguetown/boots
+	wrists = /obj/item/clothing/wrists/roguetown/bracers/leather
+	pants = /obj/item/clothing/under/roguetown/tights/black
+	cloak = /obj/item/clothing/cloak/cape/puritan
+	backr = /obj/item/storage/backpack/rogue/satchel/black
+	beltr = /obj/item/storage/belt/rogue/pouch/coins/rich
+	head = /obj/item/clothing/head/roguetown/puritan
+	gloves = /obj/item/clothing/gloves/roguetown/leather
+	beltl = /obj/item/rogueweapon/sword/rapier
+	backpack_contents = list(/obj/item/storage/keyring/puritan = 1, /obj/item/lockpickring/mundane = 1, /obj/item/rogueweapon/huntingknife/idagger/silver)
+	if(H.mind)
+		H.mind.adjust_skillrank(/datum/skill/combat/swords, 4, TRUE)
```

Comment:
4 in a combat stat is so good that only like three adventurers get it. this class has 4 in three combat stats. seems a bit much.

Reply:
Inquisitor isn't equivalent to adventurer, it isn't much when it's a singular, titular role. It shouldn't be rivaled by John Adventurer running Warrior